1.1.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+#include <conio.h>
+
+#define N 10
+
+void test()
+{
+    int numbers[N];
+    int i, j;
+
+    for (i = 0; i < N; i++)
+    {
+        printf("numbers[%d] = ", i);

```

Comment:
```suggestion
        printf("Enter the number #%d= ", i);
```
Программа должна говорить с пользователем на человеческом языке, а не на компьютерном.
(-1 балл)